Sources/NIOSSL/SSLCertificate.swift

```diff
@@ -46,6 +46,16 @@ public class NIOSSLCertificate {
         case ipv4(in_addr)
         case ipv6(in6_addr)
     }
+    
+    public lazy var serialNumber: String = {
```

Comment:
`public lazy` is a nasty combination because it's not thread safe. In this case the computation is cheap, so we should just do this as a computed property.